vendor/sunnyvalley/src/etc/pkg/repos/SunnyValley.conf.default

```diff
@@ -2,6 +2,6 @@ SunnyValley: {
   fingerprints: "/usr/local/etc/pkg/fingerprints/SunnyValley",
   url: "https://updates.sunnyvalley.io/opnsense/repo/",
```

Comment:
How about taking this a little further and doing something like:

```suggestion
  url: "https://updates.sunnyvalley.io/opnsense/%%FLAVOR%%repo/",
```

And then doing a quick sed from the +POST_INSTALL instead of copying the respective file? This would remove the need for a .LibreSSL file and still give you URL flexibility for the future? That being said, the plugin can always be changed to be smarter about the URL assembly if required later by adding more code.

Reply:
Looks a lot sexy :) The only thing that came to my mind: repo file being a package file, this kind of modification would result in checksum mismatch, since we're modifying the repo file?

Reply:
you leave this file at .conf.default and sed-pipe it to .conf :)

Reply:
ping :)

Reply:
@fichtner , this looks neat :) Sorry for the latency. I'll update this this week.